be/node/index.ts

```diff
@@ -23,164 +26,79 @@ app.use(zip());
 app.post(
   "/musicxml-to-svg",
   upload.single("musicxml"),
-  [
-    body("pageWidth").isFloat({ gt: 0 }),
-    body("pageHeight").optional().isFloat({ min: 0 }),
-  ],
+  [body("pageWidth").isInt({ gt: MIN_PAGE_WIDTH - 1 })],
```

Comment:
```suggestion
  [body("pageWidth").isInt({ min: MIN_PAGE_WIDTH })],
```
I noticed that you used `isInt` instead of `isFloat`. I remeber that I used `isFloat` because I think on the frontend, before we call the backend, we want to get the width and sometimes the width is not a whole number.

On the other hand, if we want `isInt`, the frontend would first convert to whole number first before calling backend. Althought it makes the numbers nicer, the frontend now has the responsibility to centre it nicely (or something like that) or the sheet music might be "lopsided".

I'm not too sure what to feel here.

Reply:
The original code converted `pageWidth` and `pageHeight` to an int here:

https://github.com/max-y-huang/Schmilblick/blob/af3451d5d5327293caffc2842d247e74cf050825/be/node/index.ts#L39-L40

It should be fine.

Reply:
Rendering should also be more crips with integer sizes as to avoid aliasing.

Reply:
Sounds good